src/lib/unresolved.html

```diff
@@ -22,7 +22,7 @@
   if (window.WebComponents) {
     addEventListener('WebComponentsReady', resolve);
   } else {
-    resolve();
+    addEventListener('load', resolve);
```

Comment:
Let's use `document.readyState` iff not at least at `interactive` and then resolve on `interactive`. `load` is overly long.